layers/+lang/clojure/packages.el

```diff
@@ -269,7 +269,8 @@
   (add-hook 'cider-repl-mode-hook
             (if dotspacemacs-smartparens-strict-mode
                 #'smartparens-strict-mode
-              #'smartparens-mode)))
+              #'smartparens-mode))
+  (sp-local-pair 'clojure-mode "`" nil :actions nil))
```

Comment:
It is the next sexp after `add-hook` :). First bracket closes `if` and second closes `add-hook`, after that the `sp-local-pair`). Easier to check in full file, but `sp-local-pair` is second sexp in `clojure/post-init-smartparens`.